tests/asdf_tests/test_asdf_core.py

```diff
@@ -36,12 +36,18 @@
         WXRotation.from_rotvec(_base_rotation.as_rotvec()),
         WXRotation.from_euler(seq="xyz", angles=[10, 20, 60], degrees=True),
         WXRotation.from_euler(seq="xyz", angles=[0.2, 1.3, 3.14], degrees=False),
+        WXRotation.from_euler(seq="xyz", angles=Q_([10, 20, 60], "degree")),
+        WXRotation.from_euler(seq="xyz", angles=Q_([0.2, 1.3, 3.14], "rad")),
+        WXRotation.from_euler(seq="xyz", angles=Q_([0.2, 1.3, 3.14], "")),
```

Comment:
exactly, passing `""` indicates `dimensionless` (this would be the default to define "numers" if we ever get to the point where we enforce pint units completely)

what is a little unintuitive in my eyes is that `degree`/`rad` are also considered `dimensionsless` (see output of `Q_([10, 20, 60], "degree")).dimensionality`
So pint does not have a dimensionality of `[angle]` or similar but we could maybe add that to our unit registry if we wanted to (not sure if or how but maybe)
but that would be for later